### ISSUE TITLE
What is the reason to store file in device? 

### ISSUE THREAD

**Issue by zhangshaohu** [speechbrain/speechbrain#1291]
In interfaces.py:
        # Storing waveform in the specified device
        wavs, wav_lens = wavs.to(self.device), wav_lens.to(self.device)
Why do we need to save the audio?  It takes so many resources! I commented on all the lines but it still stores the speech file. How can we don't save the audio file?

**Comment by Gastron** [comment]
This does not save the audio in any new hard drive or anything like that. Here basically the `.to()` methods just move data from CPU-side RAM to GPU-side VRAM. This is of course absolutely necessary - the data needs to be where the computation happens.

**Comment by zhangshaohu** [comment]
Thank you for your response. I used the simple demo below
classifier = foreign_class(source="speechbrain/emotion-recognition-wav2vec2-IEMOCAP", pymodule_file="custom_interface.py", classname="CustomEncoderWav2vec2Classifier")
but every time it copied the original speech file to the local directory. Anyone experienced this?

**Comment by Gastron** [comment]
Ah. I replied a little hastily last time. The `fetch` functionality does this type of copying/symlinking, and that has turned out to be a very annoying feature for many users. However we're going to implement changes soon. See discussion e.g. in PR #1268

**Comment by zhangshaohu** [comment]
This commit solve this problem
https://github.com/stachu86/speechbrain/commit/887ed8683acc5b34663dc035361c8ddbdbc559b5

**Comment by Gastron** [comment]
Yes, it's just that we're trying to support all these different sources.
